app/javascript/application.js

```diff
@@ -1,3 +1,15 @@
-// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
+// Entry point for the build script in your package.json
 import "@hotwired/turbo-rails"
-import "controllers"
+import "./controllers"
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import store from './store';
+import Greeting from './greeting';
+
+ReactDOM.render(
+  <Provider store={store}>
+    <Greeting />
```

Comment:
- [ ] Hi @billymohajeri, you just developed an amazing app and random greetings are displayed as expected :+1: Please once more let's kindly adhere to the project requirements. There should be an App component with routes(from react-router) that lead to the Greeting component.